src/librustc/arena.rs

```diff
@@ -123,6 +123,9 @@ macro_rules! arena_types {
             [few] inferred_outlives_crate: rustc::ty::CratePredicatesMap<'tcx>,
             [] upvars: rustc_data_structures::fx::FxIndexMap<rustc_hir::HirId, rustc_hir::Upvar>,
 
+            // Interned types
+            [] tys: rustc::ty::TyS<$tcx>,
```

Comment:
Huh, so we're not using the `DroplessArena` for these anymore?

Reply:
We do, but `TyS` is not `Copy`, so we have to declare it.